### ISSUE TITLE
Application in UI has doubled icon / button of ingress

### ISSUE THREAD

**Issue by zagr0** [argoproj/argo-cd#26522]
Checklist:

- [x] I've searched in the docs and FAQ for my answer: https://bit.ly/argocd-faq.
- [x] I've included steps to reproduce the bug.
- [x] I've pasted the output of `argocd version`.

**Describe the bug**

There is doubled icon / button in UI that behaves different, first one for ingress url, second opens application

**To Reproduce**

Add application with ingress resource

**Expected behavior**

To be displayed single icon

**Screenshots**

<img width="321" height="303" alt="Image" src="https://github.com/user-attachments/assets/93ae2bc6-f2ec-48c3-abab-2ca51cf49d91" />

**Version**

```shell
v3.3.1
```

**Comment by choejwoo** [comment]
Are you suggesting merging them for UX reasons, even though they serve different purposes (one for the Ingress URL/externalURLs, one for opening the app)?

**Comment by zagr0** [comment]
I think its better to have only one for ingress in order to avoid confusion

**Comment by choejwoo** [comment]
> I think its better to have only one for ingress in order to avoid confusion

Opening app button is not only for application detail page itself, you can check annotation usage in [docs](https://argo-cd.readthedocs.io/en/stable/operator-manual/managed-by-url/).
<img width="594" height="144" alt="Image" src="https://github.com/user-attachments/assets/a92a0d1d-629c-4f3f-9dc4-9c1276fce36e" />

**Comment by zagr0** [comment]
If these two buttons have different purpose, they definitely should be visually fixed / improved
- not overlap each other
- have the same scale
- be vertically aligned
- use different icons to avoid confusion

**Comment by Hronom** [comment]
Same on `3.3.2`

> If these two buttons have different purpose, they definitely should be visually fixed / improved
> 
> * not overlap each other
> * have the same scale
> * be vertically aligned
> * use different icons to avoid confusion

Agree

**Comment by zagr0** [comment]
Relevant
